src/seedu/addressbook/AddressBook.java

```diff
@@ -1143,17 +1143,12 @@ private static String getUsageInfoForExitCommand() {
      *         UTILITY METHODS
      * ============================
      */
-
+    
     /**
-     * Removes sign(p/, d/, etc) from parameter string
-     *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
-     * @return  string without the sign
+     * Removes prefix from the given fullString if prefix occurs at the start of the string.
      */
-    
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String fullString, String prefix) {
+    	return fullString.replaceFirst(prefix, "");
```

Comment:
Shouldn't there be a validation to check if fullString.indexOf(prefix) == 0 before you actually do the replace?? If not the method behaviour contradicts the header comments.